spec/acceptance/users_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rspec_api_documentation/dsl"
+
+RSpec.resource "Users" do
+  let(:authenticated_user) do
+    create(:confirmed_user)
+  end
+
+  let(:auth_token) do
+    Knock::AuthToken.new(payload: { sub: authenticated_user.id }).token
+  end
+
+  shared_context "user parameters" do
+    parameter "first_name", <<-DESC, scope: :user, required: true
+      The first name of the user.
+    DESC
```

Comment:
There's a gem called [grape](https://github.com/ruby-grape/grape) which is designed to build APIs and it has a definition similar to this :smiley: